en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-1-all-in-one-ha.md

```
@@ -1,533 +1,830 @@
-  ```bash
-  kubectl create secret generic apim-keystore-secret --from-file=wso2carbon.jks --from-file=client-truststore.jks --from-file=wso2internal.jks -n <namespace>
-  ```
+The Helm chart mounts a Kubernetes secret named `apim-keystore-secret` as a volume into the pods. The pods will not start if this secret does not exist — you will see a `MountVolume.SetUp failed: secret "apim-keystore-secret" not found` error.
 
-> By default, this deployment uses the default keystores and truststores provided by the relevant WSO2 product.
-> For advanced details regarding managing custom Java keystores and truststores in a container-based WSO2 product deployment,
-  please refer to the [official WSO2 container guide](https://github.com/wso2/container-guide/blob/master/deploy/Managing_Keystores_And_Truststores.md).
+1. WSO2 API Manager ships with default keystores inside the Docker image. Extract them:
 
-### 1.3 Configure Internal Encryption Key (Mandatory)
+    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)